diff --git a/Makefile.obexd b/Makefile.obexd
index 2e33cbc..d5d858c 100644
--- a/Makefile.obexd
+++ b/Makefile.obexd
@@ -2,12 +2,12 @@
 if SYSTEMD
 systemduserunitdir = @SYSTEMD_USERUNITDIR@
 systemduserunit_DATA = obexd/src/obex.service
+endif
 
 dbussessionbusdir = @DBUS_SESSIONBUSDIR@
 dbussessionbus_DATA = obexd/src/org.bluez.obex.service
-endif
 
-EXTRA_DIST += obexd/src/obex.service.in obexd/src/org.bluez.obex.service
+EXTRA_DIST += obexd/src/obex.service.in obexd/src/org.bluez.obex.service.in
 
 obex_plugindir = $(libdir)/obex/plugins
 
